numbat/modules/units/misc.nbt

```diff
@@ -63,6 +63,9 @@ unit footballfield: Area = 105 m × 68 m
 # Olympic-size swimming pool (FR3)
 unit swimmingpool: Volume = 50 m × 25 m × 2 m
 
+# https://en.wikipedia.org/wiki/Smoot
+unit smoot: Length = 5 feet + 7 inch
```

Comment:
Tests are failing because `feet` and `inch` are not defined here. This module is only importing `units::si`, and I'd like to keep it that way.

Maybe let's add a new `units::humorous` module?

Reply:
That's totally on me for not actually running the tests locally :facepalm:, I'll do that